bitrise.yml

```diff
@@ -191,4 +191,4 @@ app:
 meta:
   bitrise.io:
     machine_type_id: g2.12core
-    stack: osx-xcode-13.4.x
+    stack: osx-xcode-14.0.x
```

Comment:
Note, I think this _may_ break automatic uploads to app store connect until https://github.com/fastlane/fastlane/pull/20631 is merged. Given we need to start using Xcode 14 is probably ok, though for now.

Reply:
I would bet that upstream fastlane change merges within the next day or so. Its a big change breaking a lot of folks workflows.